packages/docusaurus-theme-classic/src/theme/DocSidebar/index.js

```diff
@@ -106,8 +106,18 @@ function mutateSidebarCollapsingState(item, path) {
         items
           .map((childItem) => mutateSidebarCollapsingState(childItem, path))
           .filter((val) => val).length > 0;
+
+      // Check if the user wants the category to be expanded by default
+      const shouldExpand = item.collapsed === false;
+
       // eslint-disable-next-line no-param-reassign
       item.collapsed = !anyChildItemsActive;
+
+      if (shouldExpand) {
+        // eslint-disable-next-line no-param-reassign
+        item.collapsed = false;
+      }
+
```

Comment:
@lex111 I took what you said and this is the approach that I came up with. From the local testing I did, it seems to have the intended effects without causing bugs. Could you verify?

Reply:
Great! Could we make as example expanded  top-level category of Docusaurus? https://github.com/facebook/docusaurus/blob/master/website/sidebars.js#L10

Reply:
Sure! I'll add that now